includes/managers/icons.php

```diff
@@ -256,6 +256,12 @@ public static function get_font_icon_svg( $icon, $attributes = [] ) {
 			self::$font_icon_svg_symbols[ $icon_data['key'] ] = $icon_data;
 		}
 
+		if ( array_key_exists( 'class', $attributes ) && ! is_array( $attributes['class'] ) ) {
+			$class_name = $attributes['class'];
+
+			$attributes['class'] = [ $class_name ];
```

Comment:
You can do it in one line: 
```suggestion
			$attributes['class'] = [ $attributes['class'] ];
```

Reply:
Done.